pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -155,6 +155,7 @@ def __init__(
         self.period = period
         self.last_global_step_saved = -1
         self.prefix = prefix
+        self.current_score = 0
```

Comment:
```suggestion
        self.current_score = None
```
default to `None`?

Reply:
`best_model_score` defaults to 0. Chose it here for consistency.

Do you want me to change both to `None`?

Reply:
yes, that was my thoughts that for min 0 is already the min

Reply:
see: https://github.com/PyTorchLightning/pytorch-lightning/pull/4721/files#r525591032